src/components/themed/Fade.js

```diff
@@ -1,86 +1,54 @@
 // @flow
 
 import * as React from 'react'
-import { type AnimatedValue, Animated } from 'react-native'
+import Animated, { interpolate, useAnimatedStyle, useSharedValue, withTiming } from 'react-native-reanimated'
 
-type OwnProps = {
-  timing?: 'ease' | 'linear',
+// $FlowFixMe
```

Comment:
The whole point of having the "future-commit" was to remove this `$FlowFixMe` ;)